packages/documentation/src/stories/components/tabs/tabs.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import { Meta, StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { BADGE } from '../../../../.storybook/constants';
+
+const meta: Meta<HTMLPostTabsElement> =  {
+  title: 'Components/Tabs',
+  component: 'post-tabs',
+  render: renderTabs,
+  parameters: {
+    badge: [BADGE.BETA, BADGE.NEEDS_REVISION],
+  },
+  argTypes: {
+    activePanel: {
+      name: 'active-panel',
```

Comment:
```suggestion
      name: 'Initial panel',
      description: 'The initially active panel. Changing this value after initialization won\'t have any effect.'
```

Reply:
You're right, I wanted to highlight the fact that the component does not update and that that's intended behavior.

Reply:
I updated the description.